cypress/e2e/default/learn/header/universal-navigation.js

```diff
@@ -11,6 +11,8 @@ const selectors = {
   'navigation-list': '.nav-list',
   'toggle-button': '.toggle-button-nav',
   'language-menu': '.nav-lang-menu',
+  'exit-lang-menu': "[data-value='exit-lang-menu']",
+  'lang-menu-option': 'button.nav-lang-menu-option',
```

Comment:
Would it be possible to use a `data-test-label` for this selector as well? Something like `'nav-lang-menu-option': "[data-test-label='nav-lang-menu-option']"`.

If so, `nav-links.tsx` would need to be updated as well.

But if not, I don't think it's a big deal to use an element + class selector here.

Reply:
yeah, It could be better if we could use `data-test` selectors in the tests but there is none.
And I don't want to make changes to more files (specially .tsx)

Reply:
Sure thing, that makes sense.

There are some existing tests that use element / class selectors, so I don't see the harm in adding one more here.